Your_second_ASDF_converter.ipynb

```diff
@@ -0,0 +1,1142 @@
+    "%%writefile src/myconverters/__init__.py\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "799981ad",
+   "metadata": {},
+   "source": [
+    "And now to add the setup files. Note the change to integrations.py for get_extensions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b82c5f58",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile setup.cfg\n",
+    "[metadata]\n",
+    "name = myconverters\n",
+    "description = TODO\n",
+    "long_description = TODO\n",
+    "author = Bullwinkle\n",
+    "version='0.1.0'\n",
+    "license = BSD-3-Clause\n",
+    "\n",
+    "[options]\n",
+    "zip_safe = True\n",
+    "python_requires = >=3.6\n",
+    "setup_requires =\n",
+    "    setuptools_scm\n",
+    "install_requires =\n",
+    "    jsonschema>=3.0.2\n",
+    "    asdf>=2.8\n",
+    "    psutil>=5.7.2\n",
+    "    numpy>=1.16\n",
+    "package_dir =\n",
+    "    =src\n",
+    "packages = find:\n",
+    "\n",
+    "[options.entry_points]\n",
+    "asdf.extensions =\n",
+    "    my_extension = myconverters.integration:get_extensions\n",
+    "\n",
+    "[options.packages.find]\n",
+    "where = src"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1ebbffa3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile setup.py\n",
+    "#!/usr/bin/env python3\n",
+    "from setuptools import setup\n",
+    "\n",
+    "setup(use_scm_version=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "21c2ef73",
+   "metadata": {},
+   "source": [
+    "Dealing with the Schemas\n",
+    "------------------------------\n",
+    "The utility previously mentioned would have contructed a directory tree for us with many files populated\n",
+    "\n",
+    "Only the strictly necessary parts will be done manually, partly to minimize what has to be done, but also it isn't easy to edit files in a tutorial notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "02fb0e4b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd .."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "28402214",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir myschemas"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "308c46c1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd myschemas"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a36535a5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir src"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fd3a8d6b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir src/myschemas"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "886f2ccc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir src/myschemas/resources"
```

Comment:
This is different from the structure created by the cookiecutter template, which puts the resources directory in the repo root.

Reply:
Hmmm, we seemed to have done it this way for rad. Should we have not done that?